routing/base/astar_algorithm.hpp

```diff
@@ -47,7 +48,10 @@ class AStarAlgorithm
     return os;
   }
 
-  using TOnVisitedVertexCallback = std::function<void(TVertexType const &, TVertexType const &)>;
+  using OnVisitedVertexCallback = std::function<void(TVertexType const &, TVertexType const &)>;
+  // Callback used to check total ending to edge weight before adding edge to AStar edges queue.
```

Comment:
What do you mean by "total ending to edge weight"?

Reply:
I'll replace with `path length from start/finish to edge including edge`. Is it ok?

Reply:
Ok

Reply:
done